gdrive/gdrive.go

```diff
@@ -0,0 +1,77 @@
+package gdrive
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+func (g *GoogleDriveStorage) UploadFile(path, filename string, options map[string]interface{}) (types.UploadFile, error) {
```

Comment:
This will need to be refactored to use `fileFace interface{}` for the input argument